ProcessMessage/handler.ts

```diff
@@ -404,13 +426,25 @@ export const getProcessMessageHandler = ({
 
           if (O.isNone(maybeProfile)) {
             // the recipient doesn't have any profile yet
-            context.log.warn(`${logPrefix}|RESULT=PROFILE_NOT_FOUND`);
-            await getMessageStatusUpdater(
-              lMessageStatusModel,
-              createdMessageEvent.message.id,
-              newMessageWithoutContent.fiscalCode
-            )(MessageStatusValueEnum.REJECTED)();
+            await pipe(
+              getMessageStatusUpdater(
```

Comment:
it looks like you're reusing the result of this operation, maybe you can save some lines storing it into some `messageStatusUpdater` var

Reply:
Right, thanks. Done in [a31161b](https://github.com/pagopa/io-functions-services/pull/206/commits/a31161b0eeea31e89cc9c2dd52ccd0b6c566b7a0)